README.md

```diff
@@ -74,45 +103,53 @@ Total Flash memory (text + data + misc): 60554 bytes
 
-When the build succeeds, you have created a bootloader for your target. This bootloader defines the update file to be located at `/sd/mbed-os-example-bootloader-blinky_application.bin`. This is the binary bootloader flashes and then jumps to.
+In this example, the update binary for the bootloader is defined in `main.cpp`  as `mbed-os-example-bootloader-blinky_application.bin` (specified under the macro `UPDATE_FILE`). The bootloader will look for this file in the root of the SD card, flash it to memory, and then jump to the application.
 
 ## Set up application to use bootloader
 
-The next step is to build an application you can combine with your bootloader to create a loadable image. 
+The next step is to build an application you can combine with your bootloader to create a loadable image. Any application works; however, you may wish to start with the [Blinky example](https://github.com/ARMmbed/mbed-os-example-blinky).
 
-1. Update your board to use the newly created bootloader image. To do this, set the target (replace `<TARGET_NAME>` with your target name) value `bootloader_img` to the file path of the bootloader image.
+1. Instruct the build system to use the newly created padded bootloader image (named `<project-name>.bin` in the previous section). To do this, create a `target_overrides` section in your application's `mbed_app.json` file and add a key for your target (see `<TARGET_NAME>` in the example below). If `mbed_app.json` does not already exist in your application, create one.
+
+2. Set the target value `bootloader_img` in `mbed_app.json` to the file path of the padded bootloader image . For the below example, copy the padded bootloader image to the project directory.
 
 ```
+{
     "target_overrides": {
         ...
         "<TARGET_NAME>": {
-            "target.bootloader_img": "bootloader/<TARGET_NAME>.bin"
+            "target.bootloader_img": "<path to padded bootloader binary>"
         },
         ...
+}
 ```
 
-2. Invoke `mbed compile`, and specify the name of your platform and the toolchain you are using (`GCC_ARM`, `ARM`, `IAR`). For example, for the ARM Compiler 5:
+2. Invoke `mbed compile`, and specify the name of your target and the toolchain you are using (`GCC_ARM`, `ARM`, `IAR`). For example, for the ARM Compiler 5:
 
     ```
     mbed compile -m <TARGET_NAME> -t ARM
     ```
 
+After the build completes, the build system saves two binary images in the `BUILD/<TARGET_NAME>/<TOOLCHAIN_NAME>` directory. The image that contains the combined bootloader and application (suitable for flashing to the device's ROM directly) is named `<project-name>.bin`. Use the image saved with the name `<project-name>_application.bin` as the update binary (the one you place on the SD card).
```

Comment:
Query: Did you [the reader] save the image with that name? If so, please change the last sentence to "Use the image you saved with the name ..." Otherwise, please leave as is.